File: MATLAB/examples/BrockMirman_dyn.m
function out = BrockMirman_dyn(in,param)

kpp =  in(1);
kp =   in(2);
k =    in(3);
zp =   in(4);
z =    in(5);

alf = param(1);
bet = param(2);

c  = exp(z)*k^alf - kp;
cp = exp(zp)*kp^alf - kpp;
rp = alf*exp(zp)*kp^(alf-1);
out = bet*(c/cp)*rp - 1;

File: PYTHON/examples/BrockMirman_dyn.py
from __future__ import division
import numpy as np

def BrockMirman_dyn(In,param):

	kpp =  In[0]
	kp =   In[1]
	k =    In[2]
	zp =   In[3]
	z =    In[4]

	alf = param[0]
	bet = param[1]

	c  = np.exp(z)*k**alf - kp
	cp = np.exp(zp)*kp**alf - kpp
	rp = alf*np.exp(zp)*kp**(alf-1)
	Out = bet*(c/cp)*rp - 1
	return Out